### ISSUE TITLE
cant open wallet in windows. nothing happens! 

### ISSUE THREAD

**Issue by 407mike407** [CityOfZion/neon-wallet#993]
when i open the wallet it does nothing nothing at all happens. just an icon installed on my desktop with no installation files present on hard drive. the task manager shows the app running in background processes but stats show at 0 percent all the way across. ive tried re installing as well as running as admin and resetting computer. nothing is working.

**Comment by kimianiaga** [comment]
Did you get any help ????

**Comment by kimianiaga** [comment]
I tried for so many days without any luck.  But I could open my Ledger HD Wallet in other computer

**Comment by Telcontar120** [comment]
Same problem here, no loading even after re-installation.  The initial load went fine but nothing after that.

**Comment by drptbl** [comment]
@407mike407 @Telcontar120 
1. Which OS are you using?
2. Which wallet version are you trying to run?
3. Did it happen before or after update or there was no connection at all with updating the wallet (it was working fine and suddenly stopped to work)? If it broke after update, what version did you use before?
4. Do you remember if you did something specific before you closed wallet last time before it broke?
5. Are you sure that nothing is blocking your connection/wallet process from access to internet connection (firewall for example)?

**Comment by Telcontar120** [comment]
@drptbl  thanks for the reply.  Here are your answers:
1. Running Windows 10 (64 bit)
2. Running Neon v 0.2.3 (latest version)
3. This is the first version of the wallet I have ever used, so it had nothing to do with updating from a prior version.
4. I did nothing at all out of the ordinary.  I opened the wallet for the first time, it opened fine, I created a new wallet, moved in my NEO, and then closed it.  Since then I cannot get the software to open again (rebooting has no effect).  As reported in another thread, after attempting to start the wallet, Windows Task Manager shows the Neon process is running but consuming no CPU resources and minimal RAM.
5. There's no firewall and nothing should be blocking it.  I have no connectivity issues with any other software.  And even that shouldn't stop the wallet from loading to the splash screen, right?  Literally I click the icon, I get a spinning activity wheel for a second or two, and then nothing.

**Comment by drptbl** [comment]
@Telcontar120 Thanks for answer!
1. Did you try running wallet as administrator?
2. Are you using some anti-virus software?
3. Based on https://github.com/electron-userland/electron-prebuilt-compile/issues/36, do you have `ePowerEvent.exe` process in task manager? If yes, can you kill it and try to run wallet again?
4. If nothing of above worked, can you remove `C:\Users\<YOUR_USER>\AppData\Roaming\Neon` folder and try to run wallet again? Please keep in mind that it will remove all of your wallet settings (imported wallets, address book, etc, so please do a backup of this folder somewhere).

Check if there are no running processes of Neon Wallet in background (using Task Manager) before 3rd and 4th step, please.

**Comment by Telcontar120** [comment]
@drptbl 
1. I had tried running wallet with admin privileges, but that didn't help.  
2. I disabled my anti-virus software while troubleshooting this issue to make sure it wasn't the culprit, and it was not.
3. I had tried to re-install the wallet from scratch as you suggested, but that didn't work either.
4.  THIS was the culprit!  I do have an Acer computer, and the ePowerEvent.exe service is part of the built-in power management utility which starts automatically.  Once I terminate that service, Neon is able to load and operate just fine.  It's quite perplexing why these two programs would be interfering with each other in this way.  And of course it would be nice if the developers could figure out what the issue is and fix it---as the other thread notes, it's not the best solution to have to force Acer users to manually kill this task whenever they want to run their wallet software!
5. Based on my results from #4, I didn't get to this step.
